geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/Simple2CacheServerDistributedTest.java

```diff
@@ -63,11 +63,11 @@ public void doMultipleCacheServer(boolean durable) throws Exception {
     int serverPort2 = vm2.invoke(WANTestBase::createCacheServer);
 
     if (durable) {
-      vm2.invoke(Simple2CacheServerDUnitTest::setCacheClientProxyTestHook);
+      vm2.invoke(Simple2CacheServerDistributedTest::setCacheClientProxyTestHook);
     } else {
-      vm3.invoke(Simple2CacheServerDUnitTest::setClientServerObserver);
+      vm3.invoke(Simple2CacheServerDistributedTest::setClientServerObserver);
     }
-    vm3.invoke(() -> CacheClientNotifierDUnitTest.createClientWithLocator(lnPort, "localhost",
+    vm3.invoke(() -> CacheClientNotifierDistributedTest.createClientWithLocator(lnPort, "localhost",
```

Comment:
Might want to extract `createClientWithLocator` to a new helper class to untangle these two distributed tests.